src/SDK/SmartDetectors.Extensions/SmartDetectors.Extensions.csproj

```diff
@@ -10,7 +10,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Management.Monitor.Fluent" Version="1.19.0" />
-    <PackageReference Include="Microsoft.Azure.Monitoring.SmartDetectors.RuntimeEnvironment.Contracts" Version="1.0.11.1" />
```

Comment:
>Contracts [](start = 92, length = 9)

So you are planning to issue a PR in SDRE to use these 3 new nugets?

Reply:
of course